blocks/header/header.js

```diff
@@ -162,5 +158,104 @@ export default async function decorate(block) {
   const navWrapper = document.createElement('div');
   navWrapper.className = 'nav-wrapper';
   navWrapper.append(nav);
-  block.append(navWrapper);
+  return navWrapper;
+}
+
+function getUserInitials() {
+  return window.user.name.split(' ').map((name) => name.charAt(0)).join('').toUpperCase();
+}
+
+function createHeaderBar() {
+  // Create TCCC primary header bar
+  const headerBar = document.createElement('div');
+  headerBar.className = 'header-bar';
+
+  // Create language section
+  const languageSection = document.createElement('div');
+  languageSection.className = 'language-selector';
+
+  const languageButton = document.createElement('div');
+  languageButton.className = 'language-selector-button';
+  languageButton.innerHTML = `
+    <span class="language-icon country-flag-usa"></span>
+    <span class="country-name">EN-US</span>
+    <span class="down-arrow-icon"></span>
+  `;
+  languageSection.appendChild(languageButton);
+
+  // Create upload button
+  const uploadButton = document.createElement('div');
+  uploadButton.className = 'header-upload-button';
+  uploadButton.innerHTML = `
+    <a class="upload-icon">Upload</a>
+  `;
+
+  // Create help section
+  const helpSection = document.createElement('div');
+  helpSection.className = 'help-section';
+
+  const helpButton = document.createElement('div');
+  helpButton.className = 'help-section-button';
+  helpButton.innerHTML = `
+    Help
+    <span class="down-arrow-icon"></span>
+  `;
+  helpSection.appendChild(helpButton);
+
+  // Create user button (user dropdown)
+  const myAccount = document.createElement('div');
+  myAccount.className = 'my-account';
+  const myAccountButton = document.createElement('div');
+  myAccountButton.className = 'my-account-button';
+  myAccountButton.innerHTML = `
+    <div class="avatar">${getUserInitials()}</div>
+    My Account
+    <span class="down-arrow-icon"></span>
+  `;
+
+  const myAccountMenu = document.createElement('div');
+  myAccountMenu.className = 'my-account-menu';
+  myAccountMenu.innerHTML = `
+    <ul>
+      <li><a href="#" id="my-profile-link">My Profile</a></li>
+      <li><a href="#">My Rights Requests</a></li>
+      <li><a href="#">My Saved Templates</a></li>
+      <li><a href="#">My Print Jobs</a></li>
+      <li><a href="#">My Collections</a></li>
```

Comment:
So when i want to stitch collections and saves searches i need to edit this?

@alexkli @inedoviesov1 should menus be authorable (page or block) or maybe configurable (in a sheet)?